src/index.html

```diff
@@ -4,9 +4,458 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="icon" href="./images/logo.png">
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="./images/logo.png" alt="logoMoYo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Gadgets</a>
+          </li>
+          <li class="nav__item">
+
+            <a href="#" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+
+            <a href="#" class="nav__link">Photo</a>
+          </li>
+          <li class="nav__item">
+
+            <a href="#" class="nav__link">Video</a>
+          </li>
```

Comment:
Remove the empty line here
```suggestion
          <li class="nav__item">
            <a href="#" class="nav__link">Video</a>
          </li>
```